.maintain/docker/Dockerfile.x86_64-linux-gnu

```diff
@@ -16,13 +16,15 @@
 # You should have received a copy of the GNU General Public License
 # along with Darwinia. If not, see <https://www.gnu.org/licenses/>.
 
-FROM centos:7
+FROM ubuntu:16.04
 
 # for gernal linux
 
 # change mirrorlist
-RUN yum -y update && yum -y upgrade && yum -y install \
-	# tool
-	git make \
-	# compiler
-	clang gcc gcc-c++ llvm
+RUN apt update && \
+    apt upgrade -y && \
```

Comment:
This can be removed I think, and the base image would be better if we change the precise version "16.04" to "xenial".

Reply:
Will this make some os using old version such as "16.04"  unable to use the binary? @AurevoirXavier @wi1dcard

Reply:
Don't think so, since we're using a relatively lower version (ubuntu 16.xx) to build the binaries. However, I haven't ever tried it.

Reply:
"xenial" means "Ubuntu 16.04 LTS"?

Reply:
BTW, "xenial" is pretty much equal to "16.xx".

Reply:
> "xenial" means "Ubuntu 16.04 LTS"?

Correct. It locks to the specific release and won't bump to the newer major versions like 18.04, 18.10, or 20.04.